client/config.js

```diff
@@ -274,9 +276,92 @@ const ADDON_MAP = {
     }
 };
 
+const CUSTOM_COMMANDS = [
```

Comment:
Maybe in the future - make the commands get fetched from one of the rest apis, so that if commands get disabled, they don't show up on client. ALTERNATIVELY: Make a new packet id (maybe like 0x80) for commands and let them get sent from the server in that packet - you could reuse this id on the client for execution. This way we dont mess around with old diep protocol.

Reply:
those commands are just clientside ones, remote commands are fetched via the api

Reply:
the only real changes to the protocol rn are:
- s -> c 0x07 packet has a new vi (accessLevel), gets ignored by the client though, so no problems should arise here
- c -> s unused 0x06 packet has been replaced with the command execution packet